internal/util/cache/lru_cache.go

```diff
@@ -143,9 +170,10 @@ func (c *LRU) Remove(key Key) {
 	defer c.m.Unlock()
 	if e, ok := c.items[key]; ok {
 		c.evictList.Remove(e)
-		kv := e.Value.(*entry)
-		delete(c.items, kv.key)
-		if c.onEvicted != nil {
+		kv := e.Value.(*Entry)
+		delete(c.items, kv.Key)
+		c.size -= uint64(kv.Size)
```

Comment:
c.storeHelper.MeasureSize

Reply:
For measureSize may produce overhead, I hope it is used for once. So size has been assign into Entry in line 135.